docs/shell_plus.rst

```diff
@@ -309,11 +309,10 @@ If using PostgreSQL the ``application_name`` is set by default to
 SQL queries
 -------------------------
 
-It is possible to print SQL queries as they're executed in shell_plus like::
+If the configuration option DEBUG is set tu True, it is possible to print SQL queries as they're executed in shell_plus like::
```

Comment:
Could you please fix the small typo at "tu" => "to" ?
Thanks !

Reply:
Sure, sorry about that!